app/admin/order.rb

```diff
@@ -0,0 +1,12 @@
+ActiveAdmin.register Order do
+  index do
+    column :user
+    column :total
+    column "Line Items" do |order|
+      order.line_items.each do |line_item|
```

Comment:
If you are not doing anything in the block given to `each`, you can simply use `order.line_items`